src/hyperfine/benchmark.rs

```diff
@@ -63,9 +63,12 @@ pub fn time_shell_command(
     if failure_action == CmdFailureAction::RaiseError && !result.status.success() {
         return Err(io::Error::new(
             io::ErrorKind::Other,
-            "Command terminated with non-zero exit code. \
+            format!(
+                "Command terminated with non-zero exit code: {}. \
              Use the '-i'/'--ignore-failure' option if you want to ignore this. \
              Alternatively, use the '--show-output' option to debug what went wrong.",
+                result.status.code().unwrap()
```

Comment:
I'm sorry. I didn't see that `status.code()` comes as an optional. In this case, we should avoid the `.unwrap()` here. But maybe we could do something like
```
result.status.code().map_or("?".into(), |c| format!("{}", c))
```
here?

Reply:
Probably possible yea, maybe it's better to only print:

> ...with non-zero exit code.

without the `:` when its `None`. And only print the `: [value]` when there is some.

Reply:
That sounds great - thanks!

Reply:
Actually, while we are at it - why not make it even better. I just read the documentation (https://doc.rust-lang.org/std/process/struct.ExitStatus.html#method.code). It says:

*"On Unix, this will return None if the process was terminated by a signal"*

So having `None` is an actual possibility here. You can easily test it by running something like
```
hyperfine "sleep 10"
```
and then calling `killall sleep` while the benchmark is running.

So maybe we should either print:
- "The command terminated with a non-zero exit code (`X`)." if the result is `Some(X)` or
- "The command has been terminated by a signal." if the result is `None`